docs/README.md

```diff
@@ -14,35 +14,89 @@ an issue:
+  * [Using Node.js APIs](tutorial/application-architecture.md#using-node.js-apis)
+  * [Using Native Node.js Modules](tutorial/using-native-node-modules.md)
+  * [Inter-Process Communication](tutorial/application-architecture.md#)
+* [Adding Features to Your App]()
+  * [Notifications](tutorial/notifications.md)
+  * [Recent Documents](tutorial/desktop-environment-integration.md#recent-documents-windows-mac-os)
+  * [Application Progress](tutorial/progress-bar.md)
+  * [Custom Dock Menu](tutorial/desktop-environment-integration.md#custom-dock-menu-mac-os)
+  * [Custom Windows Taskbar](tutorial/windows-taskbar.md)
+  * [Custom Unity Launcher](tutorial/unity-launcher.md)
+  * [Keyboard Shortcuts](tutorial/keyboard-shortcuts.md)
+  * [Offline/Online Detection](tutorial/online-offline-events.md)
+  * [Represented File for macOS Windows](tutorial/represented-file.md)
+  * [Native File Drag & Drop](tutorial/native-file-drag-drop.md)
+* [Application Accessibility](tutorial/accessibility.md)
+  * [Spectron](tutorial/accessibility.md#spectron)
+  * [Devtron](tutorial/accessibility.md#devtron)
+  * [Enabling Accessibility](tutorial/accessibility.md#enabling-accessibility)
 * [Application Distribution](tutorial/application-distribution.md)
-* [Mac App Store Submission Guide](tutorial/mac-app-store-submission-guide.md)
-* [Windows Store Guide](tutorial/windows-store-guide.md)
-* [Snapcraft Guide](tutorial/snapcraft-guide.md)
-* [Application Packaging](tutorial/application-packaging.md)
-* [Using Native Node Modules](tutorial/using-native-node-modules.md)
-* [Debugging Main Process](tutorial/debugging-main-process.md)
-* [Using Selenium and WebDriver](tutorial/using-selenium-and-webdriver.md)
-* [DevTools Extension](tutorial/devtools-extension.md)
-* [Using Pepper Flash Plugin](tutorial/using-pepper-flash-plugin.md)
-* [Using Widevine CDM Plugin](tutorial/using-widevine-cdm-plugin.md)
-* [Testing on Headless CI Systems (Travis, Jenkins)](tutorial/testing-on-headless-ci.md)
+  * [Mac App Store](tutorial/mac-app-store-submission-guide.md)
+  * [Windows Store](tutorial/windows-store-guide.md)
+  * [Snapcraft](tutorial/snapcraft.md)
+  * [Supported Platforms](tutorial/supported-platforms.md)
+* [Application Security](tutorial/security.md)
+  * [Reporting Security Issues](tutorial/security.md#reporting-security-issues)
+  * [Chromium Security Issues and Upgrades](tutorial/security.md#chromium-security-issues-and-upgrades)
+  * [Electron Security Warnings](tutorial/security.md#electron-security-warnings)
+  * [Security Checklist](tutorial/security.md#checklist-security-recommendations)
+* [Application Updates](tutorial/updates.md)
+  * [Deploying an Update Server](tutorial/updates.md#deploying-an-update-server)
+  * [Implementing Updates in Your App](tutorial/updates.md#implementing-updates-in-your-app)
+  * [Applying Updates](tutorial/updates.md#applying-updates)
+* [Application Debugging](tutorial/application-debugging.md)
```

Comment:
I know you're going off the list I wrote here, but I kinda think the debugging section should come before distribution, security, and updates. And maybe we should give it a name that includes testing, like "Application Testing and Debugging".